evm/bridge-token-factory/contracts/BridgeTokenFactoryWormhole.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.24;
+
+import {BridgeTokenFactory} from "./BridgeTokenFactory.sol";
+import "./Borsh.sol";
+
+interface IWormhole {
+    function publishMessage(
+        uint32 nonce,
+        bytes memory payload,
+        uint8 consistencyLevel
+    ) external payable returns (uint64 sequence);
+}
+
+contract BridgeTokenFactoryWormhole is BridgeTokenFactory {
```

Comment:
And also we should generate event on NewBridgeToken: https://github.com/Near-One/omni-bridge/blob/main/near/omni-types/src/prover_result.rs#L23

Reply:
Thanks, added a39d2b1af1d04b5493db58b8a6299d8cd900c0aa